corehq/apps/export/models/new.py

```diff
@@ -1240,8 +1240,13 @@ def _process_app_build(cls, current_schema, app, form_xmlns):
 
         case_updates = form.get_case_updates(form.get_module().case_type)
         xform = form.wrapped_xform()
+        if isinstance(form.actions, AdvancedFormActions):
+            open_case_actions = form.actions.open_cases
+        else:
+            open_case_actions = form.actions.subcases
+
         repeats_with_subcases = {
-            subcase_action for subcase_action in form.actions.subcases
+            subcase_action for subcase_action in open_case_actions
```

Comment:
i actually don't think `get_open_subcase_actions` is what i'm looking for since in an advanced forms you can seemingly open a case with a repeat group in another module without a case_index. as i'm typing that, it sounds weird, but that's what i was seeing in my app. totally possible i'm misunderstanding something. i'm not that well versed in the abilities of advanced forms/modules